### PR TITLE
Implement systray integration

### PR DESCRIPTION
**Pull request by mpotthoff**
I'm using this a lot and actually had the same problems as others in #16. So I just implemented systray support myself. I'm open for any improvements you might have. I could also hide it behind a `--systray` flag if you want.

### CODE REVIEW

**Comment by benpye** [comment]
I should really turn off that AppVeyor build now 🙂

If you could put this behind a `--systray` flag that would be great.

**Comment by benpye** [comment]
Another comment, is checking in the icon.go the correct way to go here - I haven't tried but is there a way to tie this together with go generate so we just keep the source icon file?

**Comment by mpotthoff** [comment]
Okay so I added a systray flag and switched to go generate. Oh and I also added some documentation on how to build ;) Anything else that you would like to see changed?

**Comment by benpye** [comment]
Merged 👍

File: Readme.md [hunk 1]
```diff
@@ -42,6 +46,17 @@ Grab the latest release on the [releases page](https://github.com/benpye/wsl-ssh
...
 
+To create the assets.go run:
+```
+go get -u github.com/jteeuwen/go-bindata/...
+go generate
+```
+
+To create a build without a console window:
+```
+go build -ldflags -H=windowsgui
```
Comment (schrej) [inline Readme.md]: I guess the release doesn't include this?

**Pull request merged**